bin/list-all

```diff
@@ -12,12 +12,11 @@ cmd="$cmd $releases_path"
 function sort_versions() {
     sed 'h; s/[+-]/./g; s/.p\([[:digit:]]\)/.z\1/; s/$/.z/; G; s/\n/ /' \
     | LC_ALL=C sort -t. -k 1,1 -k 2,2n -k 3,3n -k 4,4n -k 5,5n \
-    | awk '{print $2}' \
-    | paste -s -d" " -
```

Comment:
Why this line was removed? 🤔

Reply:
To be honest, I wasn’t clear what it was doing before, but it seems to work without it when I was testing it locally.

Reply:
Ah, I see what it was doing now. Does asdf need the output to be on a single line?

Reply:
Ah, I read the documentation, it does need to be on the same line. 

I can fix this tonight. 

Good catch, thanks for the review!